linuxbrew-common.sh

```diff
@@ -22,30 +22,52 @@ fi
 
 declare -i -r ABS_PATH_LIMIT=85
 
-get_brew_link() {
```

Comment:
`linuxbrew-copy.sh` is also using `linuxbrew-common.sh`, so it needs to be updated as well.